src/js/controller/setup-steps.js

```diff
@@ -25,15 +25,15 @@ define([
             },
             LOAD_YOUTUBE : {
                 method: _loadYoutube,
-                depends: ['LOAD_PLAYLIST']
+                depends: ['LOAD_POLYFILLS']
             },
             LOAD_SKIN : {
                 method: _loadSkin,
-                depends: []
+                depends: ['LOAD_POLYFILLS']
```

Comment:
I'm pushing for the idea that the whole app waits for polyfills - that way we don't need to worry about moving around dependencies as we change around functionality.